model/src/pyrenew/metaclass.py

```diff
@@ -339,6 +341,9 @@ def run(
                 nuts_args=nuts_args,
                 mcmc_args=mcmc_args,
             )
+        if rng_key is None:
+            rand_int = np.random.randint(0, 100000)
```

Comment:
```suggestion
            rand_int = np.random.randint(
                np.iinfo(np.int64).min, np.iinfo(np.int64).max
            )
```

For extra randomness